app/helpers/helpers.rb

```diff
@@ -0,0 +1,12 @@
+require 'sinatra/base'
+
+class Helpers
```

Comment:
The typical place for helpers in a SInatra app is in a `helpers` black in ApplicationController. When done this way, you will be able to reference the session without passing it around!